_topic_maps/_topic_map.yml

```diff
@@ -2652,8 +2652,8 @@ Topics:
     File: otel-configuration-of-instrumentation
   - Name: Sending traces and metrics to the Collector
     File: otel-sending-traces-and-metrics-to-otel-collector
-  - Name: Sending metrics to the monitoring stack
-    File: otel-config-send-metrics-monitoring-stack
+  - Name: Configuring metrics for the monitoring stack
```

Comment:
This sounds ambiguous to me. I would consider changing this to either exporting (from the in-cluster stack) or receiving. The phrasing should probably align with the one above `Sending traces and metrics to the Collector`

Reply:
@pavolloffay, I suggest checking the preview of the page: https://73313--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-configuring-metrics-for-monitoring-stack.